panic when using "observe.New" on a CLI without "MonitoringFlags"

I wrote a CLI that uses "observe.New" and it causes a panic if I'm only using `cmd.LogFlags` instead of `cmd.MonitoringFlags`

```go
var flags = cmd.Flags{
	// some custom flags
}.Merge(cmd.LogFlags)

func main() {
    app := cli.Command{
    	Name:    "cli",
    	Version: version,
    	Action:  runCli,
    	Flags:   flags,
    	Suggest: true,
    }
    if err := app.Run(ctx, os.Args); err != nil { ...}
}

func runCli(ctx context.Context, cmd *cli.Command) error {
       obsvr, err := observe.New(ctx, cmd, "tocli", &observe.Options{})
}
```

This will result in the following panic:
```shell
panic: non-positive interval for NewTicker

goroutine 20 [running]:
time.NewTicker(0x0?)
        /Users/muc/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.24.3.darwin-arm64/src/time/tick.go:38 +0xc0
github.com/hamba/statter/v2.(*registry).runReportLoop(0x140003181b0, 0x0?)
        /Users/muc/go/pkg/mod/github.com/hamba/statter/v2@v2.6.1/registry.go:40 +0x5c
created by github.com/hamba/statter/v2.newRegistry in goroutine 1
        /Users/muc/go/pkg/mod/github.com/hamba/statter/v2@v2.6.1/registry.go:32 +0x250
```

If I'm using `Merge(cmd.MonitoringFlags)` instead, there is no panic. But for the CLI I don't necessarily need Stats and Tracing.

--- a/stats.go
+++ b/stats.go
@@ -19,6 +19,8 @@ import (
 	"github.com/urfave/cli/v3"
 )
 
+const defaultStatsInterval = time.Second
+
 // Stats flag constants declared for CLI use.
 const (
 	FlagStatsDSN      = "stats.dsn"
@@ -42,7 +44,7 @@ var StatsFlags = Flags{
 		Name:     FlagStatsInterval,
 		Category: CategoryStats,
 		Usage:    "The frequency at which the stats are reported.",
-		Value:    time.Second,
+		Value:    defaultStatsInterval,
 		Sources:  cli.EnvVars(strcase.ToSNAKE(FlagStatsInterval)),
 	},
 	&cli.StringFlag{
@@ -67,6 +69,9 @@ func NewStatter(cmd *cli.Command, log *logger.Logger, opts ...statter.Option) (*
 	}
 
 	intv := cmd.Duration(FlagStatsInterval)
+	if intv <= 0 {
+		intv = defaultStatsInterval
+	}
 	prefix, tags := statsWith(cmd)
 
 	opts = append(opts, statter.WithPrefix(prefix), statter.WithTags(tags...))

--- a/stats_test.go
+++ b/stats_test.go
@@ -11,7 +11,7 @@ import (
 	"github.com/urfave/cli/v3"
 )
 
-func TestNewStats(t *testing.T) {
+func TestNewStatter(t *testing.T) {
 	log := logger.New(io.Discard, logger.LogfmtFormat(), logger.Error)
 
 	tests := []struct {

